src/parser/Sentence.h

```diff
@@ -26,96 +27,15 @@ class Sentence {
     virtual ~Sentence() {}
     virtual std::string toString() const = 0;
 
-    enum class Kind : uint32_t {
-        kUnknown,
-        kExplain,
-        kSequential,
-        kGo,
-        kSet,
-        kPipe,
-        kUse,
-        kMatch,
-        kAssignment,
-        kCreateTag,
-        kAlterTag,
-        kCreateEdge,
-        kAlterEdge,
-        kDescribeTag,
-        kDescribeEdge,
-        kCreateTagIndex,
-        kCreateEdgeIndex,
-        kDropTagIndex,
-        kDropEdgeIndex,
-        kDescribeTagIndex,
-        kDescribeEdgeIndex,
-        kRebuildTagIndex,
-        kRebuildEdgeIndex,
-        kDropTag,
-        kDropEdge,
-        kInsertVertices,
-        kUpdateVertex,
-        kInsertEdges,
-        kUpdateEdge,
-        kShowHosts,
-        kShowSpaces,
-        kShowParts,
-        kShowTags,
-        kShowEdges,
-        kShowTagIndexes,
-        kShowEdgeIndexes,
-        kShowUsers,
-        kShowRoles,
-        kShowCreateSpace,
-        kShowCreateTag,
-        kShowCreateEdge,
-        kShowCreateTagIndex,
-        kShowCreateEdgeIndex,
-        kShowTagIndexStatus,
-        kShowEdgeIndexStatus,
-        kShowSnapshots,
-        kShowCharset,
-        kShowCollation,
-        kDeleteVertices,
-        kDeleteEdges,
-        kLookup,
-        kCreateSpace,
-        kDropSpace,
-        kDescribeSpace,
-        kYield,
-        kCreateUser,
-        kDropUser,
-        kAlterUser,
-        kGrant,
-        kRevoke,
-        kChangePassword,
-        kDownload,
-        kIngest,
-        kOrderBy,
-        kShowConfigs,
-        kSetConfig,
-        kGetConfig,
-        kFetchVertices,
-        kFetchEdges,
-        kBalance,
-        kFindPath,
-        kLimit,
-        kGroupBy,
-        kReturn,
-        kCreateSnapshot,
-        kDropSnapshot,
-        kAdminJob,
-        kGetSubgraph,
-    };
-
-    Kind kind() const {
+    SentenceKind kind() const {
```

Comment:
where is the sentenceKind defined?

Reply:
common repo

Reply:
So move Sentence::Kind to common's SentenceKind ?

Reply:
It's not convenient when we add new sentence like `Match`, so i think it's not a good practice to update thrift interfaces frequently。